### ISSUE TITLE
Importing Decks - Incorrect Format?

### ISSUE THREAD

**Issue by OnlyaJedi** [extesy/DeckTracker#50]
When I try to import a deck from www.legends-decks.com I get the error "Something went wrong - Input string was not in a correct format" whether I paste the plain text from the legends-decks.com export feature or the url.  An example deck I was trying to paste:
```
###Nittany's [Legend] BEAST MODE Warrior###
3 Fiery Imp (Set1 #1821)
3 Rapid Shot (Set1 #1757)
3 Afflicted Alit (Set1 #1290)
3 Circle Initiate (Set3 #2150)
3 Shield Breaker (Set3 #2088)
3 Wind Keep Spellsword (Set1 #1567)
3 Companion Harbinger (Set3 #2217)
3 Grim Shield-Brother (Set3 #2136)
3 Morkul Gatekeeper (Set1 #1105)
1 Skyforge (Set3 #2090)
3 Young Mammoth (Set1 #1554)
3 Aela's Huntmate (Set3 #2133)
1 Garnag, Dark Adherent (Set2 #2021)
1 Reive, Blademaster (Set1 #1591)
3 Ulfric's Zealot (Set3 #2193)
3 Shadowfen Priest (Set1 #1526)
3 Sower of Revenge (Set999 #1807)
2 Child of Hircine (Set1 #1269)
2 Triumphant Jarl (Set1 #1183)
1 Aela the Huntress (Set3 #1712)
```

**Comment by extesy** [comment]
@OnlyaJedi Does it say which string?

**Comment by OnlyaJedi** [comment]
It does not, the message is simply as I typed it.

**Comment by extesy** [comment]
@OnlyaJedi Fixed in release `1.0.51`. Thank you for reporting.
